docs/developer/extending/apps/developing-apps/app-sdk/api-handlers.md

```diff
@@ -0,0 +1,148 @@
+# API Handlers
+
+Saleor Apps are meant to work in a serverless environment, where Cloud Functions are the foundations of server-side code.
+
+Currently, Saleor heavily relies on Next.js, but other platforms will be supported in the future.
```

Comment:
Might also link to protected API handlers, topics are related

Reply:
To where exactly?

Reply:
At the end of the document, after the webhooks.

```
# Protected API handlers

For validating, if incoming requests to your API originated from an application run from within Saleor Dashboard, [read more](protected-handlers).
```

Since we have a separate position in the sidebar for that, it might be redundant.